app/models/case/sar.rb

```diff
@@ -115,6 +116,47 @@ def sar?
     true
   end
 
+  def extendable?
+    max_allowed_deadline_date > external_deadline
+  end
+
+  # Original deadlines are not stored due to the way Polymorphic
+  # Cases are created. Therefore retrieving from the earliest
+  # case transition history is necessary otherwise assume current
+  # external_date is the final deadline.
+  def initial_deadline
+    sar_extensions = self.transitions
```

Comment:
is it worth adding a scope for this on case transitions or is that overkill?
just thinking we could have some kind of extended transitions scope that could collect all the sar extensions and pit extensions, that way we could make it a little more consistent

Reply:
TO DISCUSS

Reply:
Perhaps one to Roll in with any changes to `extend_for_pit` changes.

Reply:
Don't think we actually discussed this. It was just something to think about.

Reply:
I would wait until the model has more methods requiring some kind of case_transition retrieval and extract at that point.